### PR TITLE
Removing explicit status value for firmwareOSource MO.

### PR DESCRIPTION
**Pull request by msadatra**
This diff removes explicit value setting code for status field in firmwareOSource MO.
It is done this way so create API does not fail in case MO is already present in APIC.
Also, update API does not fail if MO is not present in APIC.

### CODE REVIEW

File: client/firmwareOSource_service.go [hunk 1]
```diff
@@ -57,6 +57,7 @@ func (sm *ServiceManager) CreateOSource(name string, description string, firmwar
 	rn := fmt.Sprintf("osrc-%s", name)
 	parentDn := fmt.Sprintf("uni/fabric/fwrepop")
 	firmwareOSource := models.NewOSource(rn, parentDn, description, firmwareOSourceAttr)
+	firmwareOSource.Status = ""  // this avoids API failure in case MO is already created in APIC
```
Comment (tycha-d-1-3) [inline client/firmwareOSource_service.go:60]: Why is this line necessary if NewOSource already returns a struct with Status set to empty string?  If its not needed  - remove it.

File: client/firmwareOSource_service.go [hunk 2]
```diff
@@ -80,7 +81,7 @@ func (sm *ServiceManager) UpdateOSource(name string, description string, firmwar
 	rn := fmt.Sprintf("osrc-%s", name)
 	parentDn := fmt.Sprintf("uni/fabric/fwrepop")
 	firmwareOSource := models.NewOSource(rn, parentDn, description, firmwareOSourceAttr)
-	firmwareOSource.Status = "modified"
+	firmwareOSource.Status = ""  // this avoids API failure in case MO is not created in APIC
```
Comment (tycha-d-1-3) [inline client/firmwareOSource_service.go:84]: Why is this line necessary if NewOSource already returns a struct with Status set to empty string? If its not needed - remove it.

Comment (msadatra) [reply client/firmwareOSource_service.go:84]: Will take care of it.

File: client/firmwareOSource_service.go (hunk 1 above)
Comment (msadatra) [reply client/firmwareOSource_service.go:60]: Will take care of it.
